include/astshim/base.h

```diff
@@ -43,7 +43,7 @@ using Array2D = ndarray::Array<double, 2, 2>;
 /**
 2D array of const double; typically used for lists of const points
 */
-using ConstArray2D = ndarray::Array<double const, 2, 2>;
+using ConstArray2D = ndarray::Array<const double, 2, 2>;
```

Comment:
The new way is inconsistent with [LSST convention](https://developer.lsst.io/coding/cpp_style_guide.html#the-const-keyword-should-be-listed-after-the-type-name).

Reply:
In this case one reads left to right; a similar issue comes up for shared_ptr, e.g. "shared_ptr<const foo>" is a shared pointer to a const foo. I know we've adopted this convention in real life, but I don't know if it's in the standard.

Reply:
Which standard do you mean? :confused: